### PR TITLE
fix: adding an xref about authentication as an optional prerequisite

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Add an optional prerequisite regarding authentication to a Git server to the procedure about starting a new workspace.

In the new docs, relevant pages for RHDEVDOCS-4061 are:

- https://www.eclipse.org/che/docs/next/end-user-guide/starting-a-new-workspace-with-a-clone-of-a-git-repository/
- https://www.eclipse.org/che/docs/next/end-user-guide/authenticating-yourself-to-a-git-server-from-a-workspace/

This PR aims to solve RHDEVDOCS-4061 by additionally linking the second mentioned page as an optional prerequisite in the first mentioned page.

## What issues does this pull request fix or reference?
RHDEVDOCS-4061

## Specify the version of the product this pull request applies to
7.46

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 1]
```diff
@@ -22,6 +22,7 @@ TIP: You can also use the *Git Repo URL ** field on the *Create Workspace* page
 
 * Your organization has a running instance of {prod-short}.
 * You know the FQDN URL of your organization's {prod-short} instance: `pass:c,a,q[{prod-url}]`.
+* Optional: xref:authenticating-yourself-to-a-git-server-from-a-workspace.adoc[Authentication to the Git server] is configured.
```
Comment (Srivaralakshmi) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]: Rewrite for parallelism. The rest of the sentences are in the present tense. So ensure you follow the guideline given below to rewrite the prerequisites accordingly.

Please refer to: https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@max-cx PTAL at my comment. Thanks!

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc (hunk 1 above)
Comment (max-cx) [reply modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]: @Srivaralakshmi, thank you. `Authentication to the Git server is configured` also appears to be in the present tense. What do you mean by `parallelism` on this line?

Comment (Srivaralakshmi) [reply modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]: The structure of the sentence. The rest starts with You and Your, 2nd person pronouns.

Comment (Srivaralakshmi) [reply modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]: Try something like: You have the authentication configured to the Git server.

Comment (max-cx) [reply modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]: Thank you for the call. Rewriting it.

**Review by Srivaralakshmi — APPROVED** [review]
@max-cx Thanks for the changes. Approved!

**Review by amisevsk — APPROVED** [review]

**Review by rkratky — APPROVED** [review]

**Pull request merged**